crates/usvg/src/parser/clippath.rs

```diff
@@ -57,9 +57,7 @@ pub(crate) fn convert(
         clip_path = convert(link, state, object_bbox, cache);
 
         // Linked `clipPath` must be valid.
-        if clip_path.is_none() {
-            return None;
-        }
+        clip_path.as_ref()?;
```

Comment:
I don't think this is better. If you do want to change this, I'd say:
```rust
// Bail out if the linked clip path is invalid
let clip_path_attr = convert(...)?;
clip_path = Some(clip_path_attr);
```
But I think the original code is *fine*...